static/js/compose.js

```diff
@@ -684,27 +684,26 @@ exports.initialize_pills = function () {
     });
 
     exports.pills.private_message_recipient.onPillCreate(function (value, reject) {
+        if (page_params.realm_is_zephyr_mirror_realm) {
+            // Zulip/Zephyr mirroring system means that you can
+            // message users who don't have an account in the Zulip side
+            // of the Zephyr mirroring system (yet; one will be
+            // automatically created if you actually send your message).
+            return { key: value, value: value };
+        }
```

Comment:
So, this is probably fine in that the pills would not be a regression with your moving this code, but the UX for the Zephyr model would be better if we did proper pills for users who do exist in the `people.js` data set.